src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const todoList = [
+  {
+    id: 1,
+    title: "Finish homework", 
+  },
+  {
+    id: 2,
+    title: "Go to park",
+  },
+  {
+    id: 3,
+    title: "Cook",
+  },
+];
+
+function App() {
+  return (
+    <div>
+      <h1>Todo List</h1>
+      <ul>{todoList.map(function(item){
```

Comment:
Good work. One 🎨 stylistic change is to put a new line between your `<ul>` element and the `{todoList.map(...)`. Just makes the code a bit more readable!

eg: 

```js
<ul>
    {todoList.map{function(item){
```